.rubocop.yml

```diff
@@ -102,6 +102,10 @@ Style/DotPosition:
   - leading
   - trailing
 
+Style/IfUnlessModifier:
```

Comment:
I think I agree with turning this off if this is it: https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier

but what does this have to do with max line length?